<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormOffre</class>
 <widget class="QWidget" name="FormOffre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>473</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>867</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>867</width>
    <height>473</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_stat">
     <property name="geometry">
      <rect>
       <x>411</x>
       <y>370</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Statistiques</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>30</y>
       <width>491</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>211</x>
       <y>310</y>
       <width>191</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Trier</string>
     </property>
     <widget class="QPushButton" name="pushButton_trier">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier </string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Nom offre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prix</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>411</x>
       <y>330</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Afficher les offres</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>551</x>
       <y>160</y>
       <width>281</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Modifier offre</string>
     </property>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Id offre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom offre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>37</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_idoffre">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nomoffre_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_decription_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prix_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifier">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>modifer</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>0</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Decouvrir les offrs du notre magasin:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>551</x>
       <y>360</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Supprimer offre</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_idoffre2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Id offre</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>310</y>
       <width>201</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>rechercher offre</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_recherche">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rechercher">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>55</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Nom offre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prix</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>410</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>551</x>
       <y>0</y>
       <width>281</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajouter Offre</string>
     </property>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom Offre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Descrption</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>37</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nomoffre">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_decription">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prix">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajouter_offre">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>551</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Espace Client</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Afficher les clients</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>60</y>
       <width>201</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>rechercher client</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_recherche_client">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rechercher_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_recherche">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>55</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Nom_utulisateur</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mail</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>190</y>
       <width>191</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Trier</string>
     </property>
     <widget class="QPushButton" name="pushButton_trier_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier </string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_tri">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Nom_utulisateur</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mail</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>340</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Supprimer client</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_idclient">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Id client</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
